Dockerfile

```diff
@@ -12,6 +12,8 @@ RUN ["/bin/bash", "-c", "/cromwell/docker/install.sh /cromwell"]
 RUN mkdir /etc/service/cromwell
 ADD docker/run.sh /etc/service/cromwell/run
 
+COPY src/main/resources/reference.conf /etc/cromwell.conf
+
```

Comment:
Each Docker command you have adds a new layer to the image, so it's generally a good idea to limit the number of layers.  Try adding the `cp` to the RUN command above:

```docker
RUN mkdir /etc/service/cromwell && cp /cromwell/src/main/resources/reference.conf /etc/cromwell.conf
```